src/components/Accordion/test.js

```diff
@@ -0,0 +1,132 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import ExpansionPanel from '../ExpansionPanel';
+import Accordion from './';
+
+describe('Accordion', () => {
+  it('renders its children', () => {
+    const wrapper = shallow(
+      <Accordion>
+        <ExpansionPanel>
+          <ExpansionPanel.Header />
+        </ExpansionPanel>
+      </Accordion>
+    );
+
+    expect(wrapper.children()).toHaveLength(1);
+  });
+
+  it('can be fully styled', () => {
+    const wrapper = shallow(
+      <Accordion className="foo" style={{ bar: 'bar' }} />
+    );
+
+    expect(wrapper.hasClass('foo')).toEqual(true);
+    expect(wrapper.props().style).toEqual({ bar: 'bar' });
+  });
+
+  it('every panel will be collapsed by default', () => {
+    const wrapper = shallow(
+      <Accordion>
+        <ExpansionPanel>
+          <ExpansionPanel.Header>Header 1</ExpansionPanel.Header>
+          <ExpansionPanel.Body>Body 1</ExpansionPanel.Body>
+        </ExpansionPanel>
+        <ExpansionPanel>
+          <ExpansionPanel.Header>Header 2</ExpansionPanel.Header>
+          <ExpansionPanel.Body>Body 2</ExpansionPanel.Body>
+        </ExpansionPanel>
+      </Accordion>
+    );
+    wrapper.children().forEach(panel => {
+      expect(panel.prop('expanded')).toBe(false);
+    });
+  });
+
+  it('`expanded` prop passed to panels should be overrided', () => {
+    const wrapper = shallow(
+      <Accordion>
+        <ExpansionPanel expanded={true}>
+          <ExpansionPanel.Header>Header 1</ExpansionPanel.Header>
+          <ExpansionPanel.Body>Body 1</ExpansionPanel.Body>
+        </ExpansionPanel>
+        <ExpansionPanel expanded={true}>
+          <ExpansionPanel.Header>Header 2</ExpansionPanel.Header>
+          <ExpansionPanel.Body>Body 2</ExpansionPanel.Body>
+        </ExpansionPanel>
+      </Accordion>
+    );
+    wrapper.children().forEach(panel => {
+      expect(panel.prop('expanded')).toBe(false);
+    });
+  });
+
+  it('defaultExpanded` should set a panel a `expanded`', () => {
+    const wrapper = shallow(
+      <Accordion defaultExpandedPanel={1}>
+        <ExpansionPanel>
```

Comment:
Since the intent is for `ExpansionPanel` to be inside an `Accordion` every time, you should use the pattern `Accordion.ExpansionPanel` instead. You can see an example with `SideTray`/`SideTray.Footer` in `SideTray/index.js`

Reply:
I created the `Accordion` component as a wrapper that allows only an `ExpansionPanel` to be opened at the same time. You definitely are able to use `ExpansionPanel` out of the scope of an `Accordion`